dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -431,6 +433,19 @@ public Host save(Host host, User user, boolean respectFrontendRoles) throws DotS
         contentletHost.setBoolProperty(Contentlet.DISABLE_WORKFLOW, true);
         contentletHost = APILocator.getContentletAPI().checkin(contentletHost, user, respectFrontendRoles);
 
+        if (null != contentletHost.get("hostName") && null != host.get(Host.HOST_NAME_KEY) &&
+                !contentletHost.isNew() && !contentletHost.getTitle().equals(host.get(Host.HOST_NAME_KEY))) {
+
+            UpdateContainersPathsJob.triggerUpdateContainersPathsJob(
+                    host.get(Host.HOST_NAME_KEY).toString(),
+                    (String) contentletHost.get("hostName")
+            );
+            UpdatePageTemplatePathJob.triggerUpdatePageTemplatePathJob(
+                    host.get(Host.HOST_NAME_KEY).toString(),
+                    (String) contentletHost.get("hostName")
+            );
+        }
```

Comment:
Would be nice to write a test for this

Reply:
this change came from another PR :)

Reply:
Need get done the other one to merge with this one :P